src/app/components/Image/index.test.tsx

```diff
@@ -244,7 +245,7 @@ describe('Image - Canonical', () => {
 
 describe('Image - AMP pages', () => {
   it('should preload when preload is true', async () => {
-    render(<Fixture isAmp preload />);
+    render(<Fixture preload />);
```

Comment:
In all  cases where we had `isAmp`, we should now pass `isAmp: true` to the `RequestContextProvider`. I guess the easiest option would be to pass `options` props to the [customRender](https://github.com/bbc/simorgh/blob/latest/src/app/components/react-testing-library-with-providers.tsx#L95)


```suggestion
render(<Fixture preload />, { isAmp: true });
```

Reply:
I think it's OK to be explicit about the parameters we pass to the render function in each test - rather than extracting it to an `options` object shared across multiple tests.